src/Aspire.Dashboard/Components/Resize/BrowserDimensionWatcher.razor.cs

```diff
@@ -39,27 +39,24 @@ public async Task OnResizeAsync(ViewportSize viewportSize)
     {
         var newViewportInformation = GetViewportInformation(viewportSize);
 
-        if (newViewportInformation.IsDesktop != ViewportInformation!.IsDesktop || newViewportInformation.IsUltraLowHeight != ViewportInformation.IsUltraLowHeight)
+        if (newViewportInformation.IsDesktop != ViewportInformation!.IsDesktop
+            || newViewportInformation.IsUltraLowHeight != ViewportInformation.IsUltraLowHeight
+            || newViewportInformation.IsUltraLowWidth != ViewportInformation.IsUltraLowWidth)
         {
             ViewportInformation = newViewportInformation;
             DimensionManager.IsResizing = true;
-
+            DimensionManager.InvokeOnBrowserDimensionsChanged(newViewportInformation);
             await ViewportInformationChanged.InvokeAsync(newViewportInformation);
```

Comment:
Is the change in order important here? If so, add a comment saying why. That help prevents someone else accidently change it in the future.

Reply:
it is important, wrote a comment